bibblio.py

```diff
@@ -168,6 +167,116 @@ def create_content_item(self, content_item):
 
         return content_item
 
+    def delete_content_item(self, identifier):
+        content_item_id = identifier
+
+        if isinstance(identifier, Identifier):
+            if not identifier.type == Identifier.BIBBLIO_CONTENT_ITEM_ID:
+                raise TypeError('Identifier is not a Bibblio Content Item')
+
+            content_item_id = identifier.identifier
+        delete_url = self.CONTENT_ITEMS_ENDPOINT + content_item_id
+        response = HTTP.request_with_timeout(
+            'DELETE', delete_url, headers=self.default_headers,
+            allowed_response_codes=[200],
+            disallowed_response_codes=['4xx']
```

Comment:
You don't need to specify `disallowed_response_codes` since the only allowed response code is 200.